guides/doc-Administering_Red_Hat_Satellite/topics/Disabling_Weak_Encryption.adoc

```diff
@@ -24,28 +24,16 @@ To disable weak encryption for {Project}, complete the following steps:
 +
 [options="nowrap"]
 ----
-# Foreman Proxy
-foreman_proxy::tls_disabled_versions: [ '1.1' ]
-
-# Dynflow
-foreman_proxy::plugin::dynflow::tls_disabled_versions: [ '1.1' ]
-
 # Apache
 apache::mod::ssl::ssl_protocol: [ 'ALL' , '-SSLv3' , '-TLSv1' , '-TLSv1.1' , '+TLSv1.2' ]
-
-# Tomcat / Candlepin
-candlepin::tls_versions: [ '1.2' ]
-
-# QPID Dispatch
-foreman_proxy_content::qpid_router_ssl_protocols: [ 'TLSv1.2' ]
-foreman_proxy_content::qpid_router_ssl_ciphers: 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'
```

Comment:
Does the `--foreman-proxy-content-qpid-router-ssl-protocols=TLSv1.2` option replaces this config line?

Reply:
No, only the line above.

Reply:
Turns out that the default installation needs this. I think that this line should even be the default. Without it, qpid is vulnerable to SWEET32. Now I am wondering if this is sufficient. We have a qpid router, but there's also another qpid service listening. That may need a similar line.

Reply:
I will not remove these lines and return them then

Reply:
I think it's better to use the command line option, similar to the qpid router ssl protocols.

Reply:
Now this PR adds the following command:
```
# {foreman-installer} \
--foreman-proxy-content-qpid-router-ssl-protocols=TLSv1.2
```
The Bug this PR is opened for also suggests to add the option `--foreman-proxy-tls-disabled-versions=1.1` but you said it is not required, so I removed it. The qpid-router option is still there.

Reply:
The bug is correct that the command line option is `--foreman-proxy-tls-disabled-versions`. However, the need to pass that option was removed.

> I think it's better to use the command line option, similar to the qpid router ssl protocols.

What I meant was that you can pass `--foreman-proxy-content-qpid-router-ssl-ciphers '...'` which is better than setting it in `custom-hiera.yaml`.

Reply:
I do not have such an option on Satellite 6.7, I will find some Satellite 6.8 instance and check there next week.

Reply:
It should be there. did you check `--full-help`?

Reply:
My bad, the option is there under full-help. I cannot figure out how to use it, I tried the original syntax from the custom-hiera.yml and separating by commas, neither worked:
```
# satellite-installer --foreman-proxy-content-qpid-router-ssl-ciphers 'ALL, !aNULL, +HIGH, -SSLv3, !IDEA-CBC-SHA'
ERROR: Scenario (/etc/foreman-installer/scenarios.d/SLv3,.yaml) was not found, can not continue
# satellite-installer --foreman-proxy-content-qpid-router-ssl-ciphers 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'
ERROR: Scenario (/etc/foreman-installer/scenarios.d/SLv3:!IDEA-CBC-SHA.yaml) was not found, can not continue
```

Reply:
I think you may be hitting [Bug #30218](https://projects.theforeman.org/issues/30218) where `-S` (in `-SSLv3`) is interpreted as a scenario switch. The linked BZ suggests it hasn't landed in downstream yet.

Reply:
Created https://github.com/theforeman/puppet-foreman_proxy_content/pull/301 to set it out of the box. Probably doesn't help you for now though. Perhaps we should keep this in `custom-hiera.yaml` for now until we have it solved.

Reply:
Thank you, @ekohl. I have removed the config line that configures qpid tls version because this is done via the command line option, I have kept the config line that configures SSL cyphers.